ui/src/views/tools/ImportUnmanagedInstance.vue

```diff
@@ -522,6 +523,15 @@ export default {
       }
       return 'serviceofferingdetails' in this.computeOffering ? this.computeOffering.serviceofferingdetails.maxmemory * 1 : Number.MAX_SAFE_INTEGER
     },
+    getCPUSpeed () {
+      if (!this.computeOffering) {
+        return 0
+      }
+      if (this.computeOffering.cpuspeed) {
+        return this.computeOffering.cpuspeed * 1
+      }
+      return this.resource.cpuspeed * 1 || 0
```

Comment:
is the `|| 0` needed here?

Reply:
if the NaN is created from previous expression, then NaN || 0 will return 0. I think this LGTM.